README.md

```diff
@@ -50,6 +50,28 @@ function onDeviceReady() {
 cordova plugin add cordova-plugin-media
 ```
 
+### Android Permission Quirks
+
+**[WRITE_EXTERNAL_STORAGE](https://developer.android.com/reference/android/Manifest.permission#WRITE_EXTERNAL_STORAGE)**
+
+By default, this plugin no longer enables the `WRITE_EXTERNAL_STORE` permission because the permission's protection level is set to dangerous.
+
+This permission is not required for writing to the application-specific external directories.
+
+If you are trying to write to external directories that are not owned by the application, you might need to enable this permission.
```

Comment:
I don't think this is correct and will probably mislead users down the wrong path.

As of API 30 target, which cordova-android@10 does by default, `WRITE_EXTERNAL_STORAGE` does nothing.

See https://developer.android.com/about/versions/11/privacy/storage#permissions-target-11

> If your app targets Android 11, both the WRITE_EXTERNAL_STORAGE permission and the WRITE_MEDIA_STORAGE privileged permission no longer provide any additional access.

If they are targeting a lower SDK, which is not something we generally support, then they may require the write permission.

Reply:
Fixed